packages/gensx/src/utils/bundler.ts

```diff
@@ -68,9 +86,54 @@ export async function bundleWorkflow(
     // This is handled by the build.sh script
     return path.join(outDir, "dist.tar.gz");
   } catch (error) {
-    console.error("Error bundling workflow", error);
+    console.error("Error bundling workflow with Docker", error);
     console.error("Stdout:\n", stdout);
     console.error("Stderr:\n", stderr);
     throw error;
   }
 }
+
+async function bundleLocally(
```

Comment:
Did you try skipping the bundle step altogether, and doing a dynamic import? That would let us skip the ncc call altogether, at the cost of some parity with the regular build flow.

Reply:
The original implementation of the dev server, many commits ago, didn't do any bundling and just used the raw files so I'm pretty sure we can make that just fine if we like. I did this just to get closer to parity with the regular build flow. Although in the new world, the cost is much higher so not bundling would help the experience. 

Maybe we should make this a param so a user can choose to wait longer for a more "production like build" or just get the fast version?

Reply:
For posterity, jeremy and i chatted offline and decided to remove the bundle path completely; will just compile the files locally instead.